### PR TITLE
pp counter(s) by me

### PR DESCRIPTION
**Pull request by nikzz-tea**
### Gradiental

> Size: 480x100\

### CODE REVIEW

File: nikzzCounter/index.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
+let socket = new ReconnectingWebSocket("ws://127.0.0.1:24050/ws");
+let base = document.getElementById('base');
+let ppC = document.getElementsByClassName('pp');
+let ppFCC = document.getElementsByClassName('ppFC');
+let ppSSC = document.getElementsByClassName('ppSS');
+let hit100C = document.getElementsByClassName('hit100');
+let hit50C = document.getElementsByClassName('hit50');
+let hit0C = document.getElementsByClassName('hit0');
+let sb = document.getElementsByClassName('sb');
```
Comment (Dartandr) [inline nikzzCounter/index.js]: it's better approach to use getElementById() to avoid ppC[0]

**Review by Dartandr — CHANGES_REQUESTED** [review]
1) we can't approve second counter because it isn't an original creation
2) you need to add your counter to readme

**Comment by nikzz-tea** [comment]
deleted first one, renamed second one to Gradiental, updated readme

**Comment by Dartandr** [comment]
Sorry, you probably miss understood but we can't approve counter that you leaved 
image

**Comment by nikzz-tea** [comment]
oh i just didnt see the word "second" 
im just gonna close then cuz dont really wanna add first one anymore sorry

**Pull request closed without merging**